src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -464,6 +464,24 @@ public UUID getOwner()
     }
 
     /**
+     * Returns the name of the owner of this permission instance.
+     *
+     * @return Name of the owner
+     */
+    @Nullable
+    public String getOwnerName()
+    {
+        for (@NotNull final Map.Entry<UUID, Player> entry : players.entrySet())
```

Comment:
Extension of my other reply, we should be saving it in the Permissions object, which is unique to a colony. So we are just giving colonies O(1) lookup to find their owner, instead of the O(n)ish thing we have now